.github/workflows/translate.yml

```diff
@@ -74,13 +71,11 @@ jobs:
           Set-Content -Value "${{ secrets.DEPLOYMENT_SSH_KEY }}" -Path ~/.ssh/id_ed25519
           chmod 0600 ~/.ssh/id_ed25519
           if(git stash list) {
-              git stash pop
               install -d -m 0700 ~/.ssh
               Set-Content -Value "${{ secrets.DEPLOYMENT_SSH_KEY }}" -Path ~/.ssh/id_ed25519
               chmod 0600 ~/.ssh/id_ed25519
               git config user.name "ownClouders"
               git config user.email "devops@owncloud.com"
-              git add translations/
```

Comment:
Without those there will be nothing to commit.
Please fix the add step and keep the stash pop

Reply:
Pushed my changes, please have a look at the new commit